File: scripts/with_test_translation_outputs/Translation/Avatar/WizardLMTeam/WizardCoder-33B-V1.1/Python/32e2d04029db9d731e20e67653c98bbd2613e7b9/Experiment_Results/intermediate/SR/Translation/use_test/WizardCoder-33B-V1.1/Avatar/Python/Java/codeforces_203_B.java
<｜begin▁of▁sentence｜>Below is an instruction that describes a task. Write a response that appropriately completes the request.

### Instruction:
Translate the following Python code to Java and enclose your solution inside ```java and ```.
A sample test case is provided below:

Test input:
2 4
2 1
1 2
1 1
2 2

Expected output:
-1


```
def solve_case ( ) :
    n , m = map ( int , input ( ).split ( ) )
    count = [ None ] * n
    for i in range ( n ) :
        count [ i ] = [ 0 ] * n
    ans = - 1
    for k in range ( m ) :
        x , y = map ( int , input ( ).split ( ) )
        x -= 1
        y -= 1
        found = False
        for i in range ( x - 2 , x + 1 ) :
            for j in range ( y - 2 , y + 1 ) :
                if i >= 0 and i < n and j >= 0 and j < n :
                    count [ i ] [ j ] += 1
                    if count [ i ] [ j ] == 9 :
                        found = True
        if found :
            ans = k + 1
            break
    print ( ans )
solve_case ( )


```

### Response:
```java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int n = scanner.nextInt();
        int m = scanner.nextInt();
        int[][] count = new int[n][n];
        int ans = -1;
        for (int k = 0; k < m; k++) {
            int x = scanner.nextInt() - 1;
            int y = scanner.nextInt() - 1;
            boolean found = false;
            for (int i = x - 2; i <= x; i++) {
                for (int j = y - 2; j <= y; j++) {
                    if (i >= 0 && i < n && j >= 0 && j < n) {
                        count[i][j]++;
                        if (count[i][j] == 9) {
                            found = true;
                        }
                    }
                }
            }
            if (found) {
                ans = k + 1;
                break;
            }
        }
        System.out.println(ans);
    }
}
```

This Java code does the same thing as the Python code. It reads the input, initializes the count array, and then iterates over the input points. For each point, it increments the count of the 9 cells around the point (if they exist). If any of these cells reach 9, it sets the answer to the current step and breaks the loop. Finally, it prints the answer. <｜end▁of▁sentence｜>

File: dataset/Intermediate/Translation/Avatar/v2-Python-code/codeforces_203_B.py
from scipy.stats import ttest_ind

def solveCase():
    (n, m) = map(int, input().split())
    count = [None] * n

    def nestedLoop(i, stop, step):
        if step == 0 or (step > 0 and i >= stop) or (step < 0 and i <= stop):
            return
        count[i] = [0] * n
        nestedLoop(i + step, stop, step)
    nestedLoop(0, n, 1)
    ans = -1
    ttest_ind([63, 61, 33], [63, 90, 9])
    checkConditionOne = 243
    checkConditionTwo = 85
    for k in range(m):
        (x, y) = map(int, input().split())
        x = x - 1
        y -= 1
        found = False
        for i in range(x - 2, x + 1):
            for j in range(y - 2, y + 1):
                if i >= 0 and i < n and (j >= 0) and (j < n):
                    count[i][j] += 1
                    if count[i][j] == 9:
                        found = True
        if checkConditionOne & checkConditionTwo:
            if found:
                ans = k + 1
                break
    print(ans)
solveCase()